README.md

```diff
@@ -140,6 +140,11 @@ In addition to `fastlane`'s commands, you also have access to these `fastlane` t
 - [`scan`](https://github.com/fastlane/fastlane/tree/master/scan): The easiest way to run tests for your iOS and Mac apps
 - [`precheck`](https://github.com/fastlane/fastlane/tree/master/precheck): Check your app using a community driven set of App Store review rules to avoid being rejected
 
+## Where can fastlane run?
```

Comment:
How about "System Requirements"

And I think we could put this up under Quick Start for more visibility.